server/src/repositories/access.repository.ts

```diff
@@ -81,7 +82,8 @@ class ActivityAccess implements IActivityAccess {
     return this.albumRepository
       .createQueryBuilder('album')
       .select('album.id')
-      .leftJoin('album.sharedUsers', 'sharedUsers')
+      .leftJoin('album.sharedUsers', 'albumSharedUsers')
+      .leftJoin('albumSharedUsers.user', 'sharedUsers')
```

Comment:
this looks good, but maybe in the future we can look at renaming sharedUsers to albumUsers in the entity itself.

Reply:
Did that too for consistency